content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,519 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as "t-test", "chi-squared test" & "ANOVA" in python.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.png
+ alt: statistic hypothesis testing with python cover image
+ 
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the world of work, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+### Table of Contents
+*1.0. Prerequisites.*
+
+*2.0. Goals of the tutorial.*
+
+*3.0. Understanding hypothesis testing.*
+
+*4.0. Confidence of null hypothesis (Probability Value)*
+
+*5.0. Statistical techniques for hypothesis testing*
+
+  - *Chi-Square Test*
+  - *T-Test*
+  - *ANOVA Test*
+
+*6.0. Conclusion*
+
+*7.0. Extra Resources*
+
+### Prerequisites
+The reader of this tutorial should try meet the following criteria:
+- must have basic knowledge of python programming.
+- must have an idea or the knowledge of basic exploratory data analysis and how pandas and numpy work.
+- must have basic knowledge of statistics.
+- must be one of a data science enthusiast, newbie or professional.
+
+### Goals of the Tutorial
+At the end of this tutorial, readers should be able to 
+- understand clearly the concept of statistical hypothesis testing.
+- perform the implementation of t-test, chi-squared test & ANOVA with python and a new dataset.
+- differentiate clearly the trade-off between type 1 and type 2 error of null hypothesis.
+- explain the confidence of the null hypothesis.
+- play with basic pandas function for exploratory data analysis.
+- figure out the best case-study for "t-test", "chi-squared test" and "ANOVA test".
```

Comment:
I doubt it will make sense putting goals in one paragraph, I've never seen it when it's not listed out and i think that's the best. 
but if you want me to proceed to one paragraph tell me i will try do that

Reply:
It's better in points, you're right.